### PR TITLE
Chronograf 1.6 template variables

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

File: content/chronograf/v1.6/guides/dashboard-template-variables.md [hunk 1]
```diff
@@ -10,274 +10,378 @@ menu:
...
+<img src="/img/chronograf/v1.6/template-vars-map-dropdown.png" style="width:100%;max-width:140px;" alt="Map variable dropdown"/>
 
-##### Use the variable in a cell
-<br>
-In [cell editor mode](/chronograf/latest/guides/create-a-dashboard/#step-2-enter-cell-editor-mode), create an [InfluxQL](/influxdb/latest/query_language/) query that uses the `:tag-key:` template variable.
-The query below selects all `purchases` [values](/influxdb/latest/concepts/glossary/#field-value) in the `animals` database, the `autogen` [retention policy](/influxdb/latest/concepts/glossary/#retention-policy-rp), and the `customers` measurement.
-It [groups](/influxdb/latest/query_language/data_exploration/#group-by-tags) query results by the tag key specified by the `:tag-key:` template variable.
-The template variable requires surrounding colons (`:`) in the query.
+> If values are meant to be used as string field values, wrap them in single quote ([required by InfluxQL](/influxdb/latest/troubleshooting/frequently-asked-questions/#when-should-i-single-quote-and-when-should-i-double-quote-in-queries)). This only pertains to values. String keys do not matter.
```
Comment (russorat) [inline content/chronograf/v1.6/guides/dashboard-template-variables.md]: nevermind my previous comment

File: content/chronograf/v1.6/guides/dashboard-template-variables.md [hunk 2]
```diff
@@ -10,274 +10,378 @@ menu:
...
-The sections below describe and provide examples of each template variable type.
+[`:dashboardTime:`](#dashboardtime)  
+[`:upperDashboardTime:`](#upperdashboardtime)  
+[`:interval:`](#interval)
 
-## Template Variable Types
+### dashboardTime
+The `:dashboardTime:` template variable is controlled by the "time" dropdown in your Chronograf dashboard.
 
-### Databases
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/dashboard-template-variables.md]: For all of the template variable types, are the types plural?  That is, is the type "database" or "databases" -- I would expect the types to be singular. Types: database, measurement, field key, tag key, tag value, and CSV.

File: content/chronograf/v1.6/guides/dashboard-template-variables.md [hunk 3]
```diff
@@ -10,274 +10,378 @@ menu:
...
+## Create custom template variables
+Template variables are essentially an array of potential values used to populate parts of your cells' queries.
+Chronograf lets you create custom template variables powered by meta queries or CSV uploads that return an array of possible values.
 
-### Measurements
-Vary the target [measurement](/influxdb/latest/concepts/glossary/#measurement).
+To create a template variable:
 
-#### Example
+1. Click on the `Template Variables` button at the top of your dashboard, then `+ Add Variable`.
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/dashboard-template-variables.md]: General guidelines (not just here):
* Labels should not be in back ticks (back ticks are for text as displayed in a CLI or text as entered in forms, etcs.)
* Use bold text when referring to buttons: Click **Save**.
• Use the name of the button exactly as it appears on screen.
• Don’t list the following UI element types unless needed for clarity or navigation: button, checkbox, drop-down button, field, icon, menu, link, radio button, or window. For example, write Click **Save**, not Click the Save button.

File: content/chronograf/v1.6/guides/dashboard-template-variables.md [hunk 4]
```diff
@@ -10,274 +10,378 @@ menu:
...
+`:lower:`  
+`:zoomedUpper:`  
+`:zoomedLower:`  
 
-![Tag value template creation](/img/chronograf/v1.6/g-templates-tagvalue.png)
+## Advanced template variable usage
 
-Finally, click on the green `Save Changes` button in the top right corner and exit the template variable manager by clicking on the `X`.
+### Filtering template variables with other template variables
+[Custom meta query template variables](#custom-meta-query) allow you to filter the array of potential variable values using other existing template Variables.
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/dashboard-template-variables.md]: "template Variables" -- lowercase the "V"

File: content/chronograf/v1.6/guides/dashboard-template-variables.md (hunk 2 above)
Comment (sanderson) [reply content/chronograf/v1.6/guides/dashboard-template-variables.md]: Every reference to them in Chronograf is plural. I had the same thought, but wanted to stay consistent with what is shown in Chronograf.

File: content/chronograf/v1.6/guides/dashboard-template-variables.md [hunk 5]
```diff
@@ -10,274 +10,378 @@ menu:
...
 
-##### Use the variable in a cell
-<br>
-In [cell editor mode](/chronograf/latest/guides/create-a-dashboard/#step-2-enter-cell-editor-mode), create an [InfluxQL](/influxdb/latest/query_language/) query that uses the `:tag-value:` template variable.
-The query below selects all `purchases` [values](/influxdb/latest/concepts/glossary/#field-value) in the `animals` database, the `autogen` [retention policy](/influxdb/latest/concepts/glossary/#retention-policy-rp), and the `customers` measurement.
-It asks for data where the `species` tag key equals the value specified by the `:tag-value:` template variable.
-The template variable requires surrounding colons (`:`) in the query.
+For example, lets say you want to list all the field keys associated with a measurement, but want to be able to change the measurement:
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/dashboard-template-variables.md]: "lets" should be "let's"

File: content/chronograf/v1.6/guides/dashboard-template-variables.md [hunk 6]
```diff
@@ -10,274 +10,378 @@ menu:
...
+Value of the template variable.
 
-This is a known issue and it will be fixed in a future release.
+#### Example template variable query parameter
+```
+.../?&tempVars%5BmeasurementVar%5D=cpu
+```
+
+#### Including multiple template variables in the URL
+To chain multiple template variables as URL query parameters, included the full [pattern](#pattern-for-template-variable-query-parameters) for _**each**_ template variable.
```
Comment (stevebang) [inline content/chronograf/v1.6/guides/dashboard-template-variables.md]: "included" should be "include"

**Pull request merged**